examples/io-descriptors/README.md

```diff
@@ -0,0 +1,120 @@
+# BentoML Input/Output Types Tutorial
+
+A BentoML service supports a wide range of data types when creating a Service API. The data types can be catagorized as follows:
+- Python Standards: `str`, `int`, `float`, `list`, `dict` etc.
+- Pydantic field types: see [Pydantic types documentation](https://field-idempotency--pydantic-docs.netlify.app/usage/types/).
+- ML specific types: `nummpy.ndarray`, `torch.Tensor` , `tf.Tensor` for tensor data, `pd.DataFrame` for tabular data, `PIL.Image.Image` for
+Image data, and `pathlib.Path` for files such as audios, images, and pdfs.
+
+When creating a Bentoml service, you should use Python's type annotations to define the expected input and output types for each API endpoint. This
```

Comment:
Change all BentoML `services` to `Services`. "Service" is a special term in BentoML.

```suggestion
When creating a BentoML Service, you should use Python's type annotations to define the expected input and output types for each API endpoint. This
```